bin/coreos-install

```diff
@@ -267,7 +269,7 @@ fi
 # inform the OS of partition table changes
 blockdev --rereadpt "${DEVICE}"
 
-if [[ -n "${CLOUDINIT}" ]]; then
+if [[ -n "${CLOUDINIT}" ]] || [[ -n $COPY_NET ]]; then
```

Comment:
Please quote COPY_NET here like you do below, obsessively quoting in shell leads to fewer surprises down the road. :)